.github/workflows/dart_ci.yml

```diff
@@ -0,0 +1,63 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'test_consume_*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [ 2.4.1, stable, dev ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dart-lang/setup-dart@v0.2
+        with:
+          sdk: ${{ matrix.sdk }}
+
+      - name: Print Dart SDK version
+        run: dart --version
+
+      - id: install
+        name: Install dependencies
+        run: pub get
+
+      - name: Validate dependencies
+        run: pub run dependency_validator -i pedantic over_react_format
+        if: always() && steps.install.outcome == 'success'
```

Comment:
What's the point of `always()` here?

Reply:
Not sure 😬  - its copypasta from someone else's config I found online...

Reply:
Based on https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions it looks like it will cause this step to run even if the job has been canceled.. not sure if we need that, but I guess it doesn't hurt